sdk/tables/azure-data-tables/README.md

```diff
@@ -53,12 +51,12 @@ Your Table API account URL, subsequently identified as `<your-table-account-url>
 `http(s)://<cosmosdb-account-name>.table.cosmosdb.azure.com`.
 
 ### Authenticate the client
-Every request made to the Tables service must be authorized using a connection string, shared key credential, or shared access signature. The samples below demonstrate the usage of these methods.
+Every request made to the Tables service must be authorized using a connection string, named key credential, or Shared Access Signature. The samples below demonstrate the usage of these methods.
 
-Note: Azure Tables doesn't support Azure Active Directory (AAD) authentication.
+Note: Azure Data Tables doesn't support Azure Active Directory (AAD) authentication.
```

Comment:
Is `Data` needed here as the service itself doesn't support it? So, should it just be the service name `Azure Tables`?

Reply:
Or is the convention to use `Azure Data Tables` everywhere in reference to the service itself.

Reply:
I think it's the latter, let me just make sure of that.

Reply:
It's Azure Tables.